app/user/rest.py

```diff
@@ -478,7 +478,7 @@ def send_branding_request(user_id):
         recipient=to['email'],
         service=service,
         personalisation={
-            'email': to['email'],
+            'email': get_user_by_id(user_id=user_id).email_address,
```

Comment:
Previously the recipient and requester were filled with the same email address which resulted in this bug.

Here is what the template looks like.
![Screen Shot 2020-09-29 at 15 24 04](https://user-images.githubusercontent.com/295709/94605978-d39b7600-0267-11eb-90bf-ed74f3cf86ee.png)